integreat_cms/cms/templates/push_notifications/push_notification_list_row.html

```diff
@@ -23,7 +23,7 @@
                 {% spaceless %}
                     {% for other_language in languages %}
                         <a href="{% url 'edit_push_notification' push_notification_id=push_notification.id region_slug=request.region.slug language_slug=other_language.slug %}">
-                            <i icon-name="{% if other_language in push_notification.languages %}edit-2{% else %}plus{% endif %}"
+                            <i icon-name="{% if other_language in push_notification.languages %}pencil{% else %}plus{% endif %}"
```

Comment:
```suggestion
                            <i icon-name="{% if other_language in push_notification.languages %}pen-line{% else %}plus{% endif %}"
```
Even though `edit-2` was renamed to `pen`, I think it makes sense to use `pen-line` here for consistency reasons.